README.md

```diff
@@ -29,6 +29,22 @@ Table of contents
 
 ## Getting Started
 
+### Build Prerequisites
```

Comment:
In general I think we keep this section short and reference the preqreqs in  edgex-go since they are the same.  Down to one-liner..something akin to:
> Follow the pre-req guide located here (https://github.com/edgexfoundry/edgex-go#prerequisites)
echoing @lenny-intel comment about duplicating. Go is mentioned there as well and is also out of date mentioning 1.11.5.

Reply:
Collapsed into one liner.